flybywiresim/aircraft
# fix(a380x): Various fixes pt.2: MFD, pitch trim switch, EFIS sync for LS button

Fixes #8905 
Fixes #8915 
Fixes #8880 
Fixes #8906 

## Summary of Changes

- Check for valid XPDR code in MFD/SURV/CONTROLS page (don't allow 8 or 9)
- MFD/POSITION/NAVAIDS: Allow input of LS back course, throws format error until B/C approaches are implemented in the A380X
- Systems-Host: Enable linting, no functional changes (ignore for review)
- MFD: Fix navaid deselection
- Add cockpit PITCH TRIM switch
- Fix flickering LS scale when enabling EFIS sync in EFB

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): floridude

## Testing instructions

A380X only: 
Please verify that the above four issues have been resolved:

1. Open SD FCTL page, move cockpit PITCH TRIM switches, verify that elevator trim moves
2. Open FMS/POSITION/NAVAIDS, de-select Navaids and verify it works via the ND
3. Open FMS/POSITION/NAVAIDS, tune VOR with duplicate identifier (e.g. FRA), verify that de-duplication window (DUPLICATE NAMES or sth) opens
4. Enable EFIS sync via EFB settings -> Realism, enable LS button on CAPT EFIS CP, verify LS also shows on F/O side


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
